nodejs/webrtc-hello-again/server.js

```diff
@@ -0,0 +1,257 @@
+        "tag" : uuid.v1(),
+        "publishPermissions" : ["AUDIO"]
+      });
+
+    webRTCController.createParticipant(accountId, participantBody, 
+      function(error, createResponse, context) {
+        // console.log("participant response info: ", createResponse)
+        var body = new BandwidthWebRTC.Subscriptions({"sessionId" : getSessionId()});
+        webRTCController.addParticipantToSession(accountId, getSessionId(), createResponse.participant.id, body, 
+          function(error, response, context) {
+            // now that we have added them to the session, we can send back the token they need to join
+            console.log("sending token to web participant")
+            res.send('{"token":"'+ createResponse.token +'"}')
+        }).catch(error => {
+          console.error('Error in addParticipantToSession:', error);
+          res.send('{"status":"Failed to add participant to session"}')
+      });
+
+    }).catch(error => {
+      console.error('Error in createParticipant:', error);
+      res.send('{"status":"Failed to create Participant and get token"}')
+    });
+  }).catch(error => {
+    console.error('Error in createSession:', error);
+    res.send('{"status":"Failed to create session"}')
+  });
+});
+
+/** 
+ * Start the Phone Call
+ */
+app.get('/startPSTNCall', async (req, res) => {
+  if (getSessionId() == false){
+    console.log("No web browser in session; aborting PSTN call")
+    res.send('{"status":"Failed to start PSTN call; no session id found"}')
+    return;
+  }
+  participant_response = null;
+  console.log("about to create participant")
+  // create a participant for this browser user
+  var participantBody = new BandwidthWebRTC.Participant(
+    {
+      "tag" : uuid.v1(), 
+      "publishPermissions" : ["AUDIO"]
+    });
+
+  webRTCController.createParticipant(accountId, participantBody, 
+    async function(error, createResponse, context) {      
+      // console.log("PSTN participant response info: ", createResponse)
+      participant_response = createResponse
+      PSTNToken = createResponse.token;
+      
+      var body = new BandwidthWebRTC.Subscriptions({"sessionId" : getSessionId()});
+      console.log("Add PSTN caller to the session")
+      webRTCController.addParticipantToSession(accountId, getSessionId(), createResponse.participant.id, body, 
+        function(error, response, context) {
+          // console.log("addParticipantToSession response:", response)         
+      }).catch(error => {
+        console.error('Error in addParticipantToSession:', error);
+        res.send('{"status":"Failed to add PSTN call to session"}')
+      });
+
+      var body = new BandwidthVoice.ApiCreateCallRequest({
+        "from": process.env.FROM_NUMBER,
+        "to": process.env.OUTBOUND_PHONE_NUMBER,
+        "applicationId": process.env.VOICE_APPLICATION_ID,
+        "answerUrl": process.env.BASE_CALLBACK_URL + 'callAnswered',
+        "answerMethod": "POST",
+        "callTimeout": 10
```

Comment:
This is pretty short. It only gave my phone 2 rings to answer. I would just remove this and let it use the default of 30 seconds here.